### ISSUE TITLE
IntelliJ build broken

### ISSUE THREAD

**Issue by FracturedCode** [Emonadeo/AutoRun#5]
I was attempting to update this mod to 1.17, but couldn't get `net.minecraft` to resolve for some reason. I switched back over to the 1.14 branch as a benchmark and it is also broken. Recloning the repository gives the same results. When you try to run the client, the error is
> [main] ERROR FabricLoader - Could not find valid game provider!

**Comment by emonadeo** [comment]
I just tried to reproduce this by cloning a fresh copy of 1.16. I had to update some versions and paths, but I saw that you did the same in your forked version.

One thing that changed though, is that you don't call `gradle idea` anymore. In fact the wiki explicitely states now that it breaks the development environment:

From https://fabricmc.net/wiki/tutorial:setup:
> **NOTE:** Don't run `./gradlew idea` as it messes up with gradle and breaks develop environment.

I didn't know this myself until now. Will update the readme.
Did you by any chance call `gradle idea`? If so this might be the problem here.

**Comment by FracturedCode** [comment]
That's a good tip, I did use `./gradlew idea`, but after further testing, it's not related to my issue. I have determined that my issue is regarding *any* 1.17 fabric projects. I even set up a fresh install of intellij on a VM and the issue *still* persists. I must be clicking some buttons wrong, or there's a bug that's surfacing with my specific workflow. I tried following the setup wiki to the letter. I have determined the issue is not exclusive to this project, so I will close the issue. I'll still try to fix intellij and I'll submit a PR if I get it done.
